src/Cache/BrowscapCache.php

```diff
@@ -168,8 +174,14 @@ public function setItem(string $cacheId, $content, bool $withVersion = true) : b
             $cacheId .= '.' . $this->getVersion();
         }
 
-        // Save and return
-        return $this->cache->setItem($cacheId, $data);
+        try {
+            // Save and return
+            return $this->cache->set($cacheId, $data);
+        } catch (InvalidArgumentException $e) {
+            // do nothing here
```

Comment:
Let's not catch and ignore exceptions; let them bubble up

Reply:
done